### PR TITLE
Editor Inactivity Update

### PR DESCRIPTION
**Pull request by le-sun**
Changed contribution logic to at least 3 uploaded papers or comments
Updated editor inactivity text

image

### CODE REVIEW

File: src/templates/editor_inactivity.html [hunk 1]
```diff
@@ -77,17 +77,26 @@
               </p>
               <p class="header-message">
                 We haven't seen any activity from your account this past week.
-                Editors are expected to contribute at least one paper and comment per week.
+                Editors are expected to contribute at least three meaningful contributions per week.
```
Comment (lightninglu10) [inline src/templates/editor_inactivity.html:80]: What are 3 meaningful contributions? 

What are the exact requirements here?

Comment (le-sun) [reply src/templates/editor_inactivity.html:80]: @lightninglu10 
In the slack - [Meaningful contributions include:](https://researchhub-community.slack.com/archives/G01DP8EMCGN/p1646078198722159?thread_ts=1646056936.540369&cid=G01DP8EMCGN)
```
(1) upload (paper) + summary (comment),or a
(2) response to another user (comment)
```

**Review by yattias — APPROVED** [review]

File: src/templates/editor_inactivity.html (hunk 1 above)
Comment (lightninglu10) [reply src/templates/editor_inactivity.html:80]: I think we should define the meaningful contributions in this email

Comment (le-sun) [reply src/templates/editor_inactivity.html:80]: @lightninglu10 Updated 
image

Comment (lightninglu10) [reply src/templates/editor_inactivity.html:80]: the visit researchhub button looks weird -- can you make the text always white?

Comment (lightninglu10) [reply src/templates/editor_inactivity.html:80]: Could we also get a <br /> element in between Your Contributions: and the next line, same with Meaningful Contributions include:

Just to add another line of space between them?

Comment (le-sun) [reply src/templates/editor_inactivity.html:80]: @lightninglu10 Updated
image

**Pull request merged**